packages/core/src/utils/styleUtils/stylesUtils.spec.ts

```diff
@@ -1,66 +1,94 @@
+      });
+
+      expect(result).toBe('400px');
     });
 
-    expect(result).toBe('400px');
-  });
+    it('should return value when value is not "auto"', () => {
+      const result = calculateNodeDefaultHeight({
+        children: [],
+        value: '456px',
+      });
 
-  it('should return value when value is not "auto"', () => {
-    const result = calculateNodeDefaultHeight({
-      children: [],
-      value: '456px',
+      expect(result).toBe('456px');
     });
 
-    expect(result).toBe('456px');
-  });
+    it('should return value if block is not a container', () => {
+      const result = calculateNodeDefaultHeight({
+        blockId: 'node-block-id',
+        children: [],
+        value: '567px',
+      });
 
-  it('should return value if block is not a container', () => {
-    const result = calculateNodeDefaultHeight({
-      blockId: 'node-block-id',
-      children: [],
-      value: '567px',
+      expect(result).toBe('567px');
     });
 
-    expect(result).toBe('567px');
-  });
+    it('should return defaultValue of "120px" when container is on "root" and has no children', () => {
+      const result = calculateNodeDefaultHeight({
+        blockId: CONTENTFUL_COMPONENTS.container.id,
+        children: [],
+        value: 'auto',
+      });
 
-  it('should return defaultValue of "120px" when container is on "root" and has no children', () => {
-    const result = calculateNodeDefaultHeight({
-      blockId: CONTENTFUL_COMPONENTS.container.id,
-      children: [],
-      value: 'auto',
+      expect(result).toBe(EMPTY_CONTAINER_HEIGHT);
     });
 
-    expect(result).toBe(EMPTY_CONTAINER_HEIGHT);
+    it('should return "100%" when container has a non-container child', () => {
+      const childNode: ExperienceTreeNode = {
+        type: 'block',
+        data: {
+          id: 'block1',
+          props: {},
+          dataSource: {},
+          unboundValues: {},
+          breakpoints: [],
+        },
+        children: [],
+      };
+
+      const result = calculateNodeDefaultHeight({
+        blockId: CONTENTFUL_COMPONENTS.container.id,
+        children: [childNode],
+        value: 'auto',
+      });
+
+      expect(result).toBe('100%');
+    });
   });
 
-  it('should return "100%" when container has a non-container child', () => {
-    const childNode: ExperienceTreeNode = {
-      type: 'block',
-      data: {
-        id: 'block1',
-        props: {},
-        dataSource: {},
-        unboundValues: {},
-        breakpoints: [],
-      },
-      children: [],
-    };
-
-    const result = calculateNodeDefaultHeight({
-      blockId: CONTENTFUL_COMPONENTS.container.id,
-      children: [childNode],
-      value: 'auto',
+  describe('createJoinedCSSRules', () => {
+    it('should join a given object of css rules into a string', () => {
+      const cssRules = {
+        background: 'green',
+        color: 'white',
+        'font-size': '1rem',
+      };
+
+      const res = createJoinedCSSRules(cssRules);
```

Comment:
to me this function name sounds very crypric. What is the Joined CSS rule and what is the joined vs not joined css rule.

Maybe we call it `toMinifiedCSS`?

Reply:
I was thinking of the function `Array.join` which turns an array into a joined string. Happy to rename it though if that makes it easier to understand. As it does creat both normal CSS and minified CSS, I would like to go with `stringifyCSSProperties` (thinking of JSON.stringify) or something similar to that. Or is `toCSS` maybe enough?